roles/base/tasks/redhat_tasks.yml

```diff
@@ -3,6 +3,7 @@
     update_cache: true
     name: '*'
     state: latest
+  ignore_errors: yes
```

Comment:
usually I have seen this with our old ansible where we used to install `pip` and manage packages using pip. pip and yum don't go well together. Our recent ansible has moved away from pip. So one thing we can may be try is removing pip from these servers and then try the new ansible again.